### ISSUE TITLE
PWR key pin usage issue - sim868

### ISSUE THREAD

**Issue by Elsaligi** [vshymanskyy/TinyGSM#460]
Hi,

I have been working with sim868 facing issues to poweron / poweroff sim868.
It has been mentioned in datasheet that  can power on SIM868 by pulling down the PWRKEY pin for at least 1 second and release.
power off -  can power off SIM868 by pulling down the PWRKEY pin for at least 1 second and release.

But as am switching on and off module by another controller and there are cases where i am requesting gsm poweron but pin is identified it as command to power off , i couldn`t able to find exact condition of module.

Is there any function available in tinygsm library for proper on/ off or check the status of module (function like ismodulealive).

Regards,

**Comment by SRGDamia1** [comment]
No, TinyGSM doesn't support setting or reading pin levels for on/off/status.

You can use the `testAT()` function to check if the module is responding.

**Comment by adrianca88** [comment]
Yes, I check my SIM808 status using testAT() (with an smaller timeout) to check if it's sleeping or not.
